### PR TITLE
Pin rq at <1.0

### PR DESCRIPTION
**Pull request by dralley**
rq 1.0 will have backwards incompatible changes that may or may not
affect us. In any case, it's expected to release soon, and it would be
bad if it broke our RC immediately after release.

[noissue]

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/en/3.0/nightly/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Comment by asmacdo** [comment]
Bonus points for making this change *before* rq is released.

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     'drf-yasg',
     'gunicorn',
     'PyYAML',
-    'rq>=0.12.0',
+    'rq<1.0',
```
Comment (goosemania) [inline setup.py]: @dralley , this allows rq to be <0.12, is it desired?
How about:
`rq ~=0.12` ?
This is the same as `rq >=0.12, rq<1.0`, imo.
See PEP-440 for more details https://www.python.org/dev/peps/pep-0440/#compatible-release

Comment (bmbouter) [reply setup.py]: This is definitely a better option thank you @goosemania. I'm posting a PR to fix this now.

Comment (bmbouter) [reply setup.py]: Oh actually this isn't merged yet I thought it was. No PR from me then.

**Pull request merged**